src/main/java/uk/gov/digital/ho/hocs/search/client/elasticsearchclient/ElasticSearchClient.java

```diff
@@ -96,6 +88,18 @@ public void update(CaseData caseData) {
         }
     }
 
+    public void update(Set<String> keys, CaseData caseData) {
+        UpdateRequest updateRequest = new UpdateRequest(index, caseData.getCaseUUID().toString());
+        Map<String, Object> documentMapper = objectMapper.convertValue(caseData, Map.class);
```

Comment:
I know that we do this in other places, but it does seem like a smell. Is Jackson really not able to handle this automatically somehow?